<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-GCKUP8E" failures="8" tests="17" name="com.example.runners.systemConfigureRunner" time="1080.580" errors="1" timestamp="2023-07-28T20:40:44 IST" skipped="0">
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="34.130"/>
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="47.699">
    <failure message="expected [] but found [User System Role added successfully.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [] but found [User System Role added successfully.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.workflow.SystemConfigure.validateUserAddedMessage(SystemConfigure.java:254)
at com.example.steps.SystemConfigureStep.validate_message_for_user_added_successfully(SystemConfigureStep.java:91)
at ✽.validate message for user added successfully(file:///F:/ComplianceTracker/CT-Automation/src/test/resources/feature/systemConfigure.feature:98)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="30.392"/>
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="86.888"/>
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="43.466"/>
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="30.984">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.fail(Assert.java:96)
at com.qa.workflow.CompanyGroup.AddValidCompanyGroup(CompanyGroup.java:160)
at com.example.steps.CompanyGroupSteps.user_enters_all_the_valid_details_in_company_group(CompanyGroupSteps.java:40)
at ✽.user enters all the valid details in company group(file:///F:/ComplianceTracker/CT-Automation/src/test/resources/feature/systemConfigure.feature:45)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="45.496">
    <failure message="expected [] but found [User System Role added successfully.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [] but found [User System Role added successfully.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.workflow.SystemConfigure.validateUserAddedMessage(SystemConfigure.java:254)
at com.example.steps.SystemConfigureStep.validate_message_for_user_added_successfully(SystemConfigureStep.java:91)
at ✽.validate message for user added successfully(file:///F:/ComplianceTracker/CT-Automation/src/test/resources/feature/systemConfigure.feature:88)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="32.264">
    <failure message="did not expect [Automation Entity] but found [Automation Entity]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: did not expect [Automation Entity] but found [Automation Entity]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failEquals(Assert.java:1041)
at org.testng.Assert.assertNotEqualsImpl(Assert.java:147)
at org.testng.Assert.assertNotEquals(Assert.java:1531)
at org.testng.Assert.assertNotEquals(Assert.java:1535)
at com.qa.workflow.Entity.addNewValidEntity(Entity.java:105)
at com.example.steps.EntitySteps.user_click_on_add_new_valid_entity_button(EntitySteps.java:51)
at ✽.user click on add new valid entity button(file:///F:/ComplianceTracker/CT-Automation/src/test/resources/feature/systemConfigure.feature:56)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="57.038">
    <error message="no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//tbody[@data-test=&quot;table-body&quot;]//tr//td[1][normalize-space()=&#039;Automation Company&#039;]//..//td[4]//a[@title=&#039;Delete&#039;]&quot;}
  (Session info: chrome=115.0.5790.110)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:25:48&#039;
System info: host: &#039;DESKTOP-GCKUP8E&#039;, ip: &#039;192.168.1.14&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.5&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 115.0.5790.110, chrome: {chromedriverVersion: 114.0.5735.90 (386bc09e8f4f..., userDataDir: C:\Windows\TEMP\scoped_dir5...}, goog:chromeOptions: {debuggerAddress: localhost:59394}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 2f7a391f706d7a34b27865bbb63e3acd
*** Element info: {Using=xpath, value=//tbody[@data-test=&quot;table-body&quot;]//tr//td[1][normalize-space()=&#039;Automation Company&#039;]//..//td[4]//a[@title=&#039;Delete&#039;]}" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//tbody[@data-test="table-body"]//tr//td[1][normalize-space()='Automation Company']//..//td[4]//a[@title='Delete']"}
  (Session info: chrome=115.0.5790.110)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'DESKTOP-GCKUP8E', ip: '192.168.1.14', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.5'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 115.0.5790.110, chrome: {chromedriverVersion: 114.0.5735.90 (386bc09e8f4f..., userDataDir: C:\Windows\TEMP\scoped_dir5...}, goog:chromeOptions: {debuggerAddress: localhost:59394}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 2f7a391f706d7a34b27865bbb63e3acd
*** Element info: {Using=xpath, value=//tbody[@data-test="table-body"]//tr//td[1][normalize-space()='Automation Company']//..//td[4]//a[@title='Delete']}
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at com.qa.workflow.CompanyGroup.deletecompanygroup(CompanyGroup.java:785)
at com.example.steps.CompanyGroupSteps.record_is__deleted_and_pop_up_will_close_for_company_group(CompanyGroupSteps.java:178)
at ✽.record is deleted and pop up will close for company group(file:///F:/ComplianceTracker/CT-Automation/src/test/resources/feature/systemConfigure.feature:159)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="277.886"/>
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="51.871"/>
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="34.523"/>
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="48.789">
    <failure message="expected [] but found [User System Role added successfully.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [] but found [User System Role added successfully.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.workflow.SystemConfigure.validateUserAddedMessage(SystemConfigure.java:254)
at com.example.steps.SystemConfigureStep.validate_message_for_user_added_successfully(SystemConfigureStep.java:91)
at ✽.validate message for user added successfully(file:///F:/ComplianceTracker/CT-Automation/src/test/resources/feature/systemConfigure.feature:119)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="85.316"/>
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="45.461">
    <failure message="expected [] but found [User System Role added successfully.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [] but found [User System Role added successfully.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.workflow.SystemConfigure.validateUserAddedMessage(SystemConfigure.java:254)
at com.example.steps.SystemConfigureStep.validate_message_for_user_added_successfully(SystemConfigureStep.java:91)
at ✽.validate message for user added successfully(file:///F:/ComplianceTracker/CT-Automation/src/test/resources/feature/systemConfigure.feature:77)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="47.455">
    <failure message="expected [] but found [User System Role added successfully.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [] but found [User System Role added successfully.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.workflow.SystemConfigure.validateUserAddedMessage(SystemConfigure.java:254)
at com.example.steps.SystemConfigureStep.validate_message_for_user_added_successfully(SystemConfigureStep.java:91)
at ✽.validate message for user added successfully(file:///F:/ComplianceTracker/CT-Automation/src/test/resources/feature/systemConfigure.feature:108)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.example.runners.systemConfigureRunner" name="runScenario" time="80.922">
    <failure message="expected [User added successfully.] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [User added successfully.] but found []
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.workflow.User.addvaliduser(User.java:209)
at com.example.steps.UserSteps.user_click_on_add_new_valid_user_button(UserSteps.java:50)
at ✽.user click on add new valid user button(file:///F:/ComplianceTracker/CT-Automation/src/test/resources/feature/systemConfigure.feature:67)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- com.example.runners.systemConfigureRunner -->
